docs/configuration.rst

```diff
@@ -1059,11 +1069,40 @@ to authenticate users for each other. All variables defined in this section are
     [|constant|_]
     (Value: ``"magpie_network"``)
 
-    .. versionadded:: 3.37
+    .. versionadded:: 3.38
 
     The name of the group created to manage permissions for all users authenticated using Magpie instances as external
     authentication providers.
 
+.. envvar:: MAGPIE_NETWORK_PEM_FILES
+
+    [:class:`str`]
+    (Default: ``${MAGPIE_ROOT}/key.pem``)
+
+    .. versionadded:: 3.38
+
+    Path to a PEM file containing a public/private key-pair. This is used to sign and verify communication sent between
+    nodes in the network.
+
+    Multiple PEM files can be specified if key rotation is desired. To specify multiple PEM files, separate each file
+    path with a ``:`` character. The first file in the list will contain the primary key and will be used to sign all
+    outgoing communication.
+
+.. envvar:: MAGPIE_NETWORK_PEM_PASSWORDS
+
+    [:class:`str`]
+    (Default: ``None``)
+
+    .. versionadded:: 3.38
+
+    Password used to encrypt the PEM files in :envvar:`MAGPIE_NETWORK_PEM_FILES`.
+
+    If multiple files require passwords, they can be listed with a ``:`` character separator (Note that this means that
+    passwords cannot contain a ``:``. An empty string will be treated the same as no password.
```

Comment:
Could require that passwords are `base64` encoded to avoid this problem.

Reply:
That seems like a lot of extra work for the admin user.

What if I just format it as a json list (like you suggested for the redirect_uris above)